src/Middleware/SpaServices.Extensions/test/Microsoft.AspNetCore.SpaServices.Extensions.Tests.csproj

```diff
@@ -1,17 +1,26 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <TestDependsOnNode>true</TestDependsOnNode>
+	<!-- Depends on npm which is not picked up on helix -->
+	<BuildHelixPayload>false</BuildHelixPayload>
```

Comment:
Fair, but let's follow the pattern [established elsewhere](https://github.com/dotnet/aspnetcore/blob/0517745c08d1bc85009f1ca41a41d77176eeea60/src/Hosting/test/FunctionalTests/Microsoft.AspNetCore.Hosting.FunctionalTests.csproj#L7) and include a comment with the GitHub issue tracking the fact that Helix doesn't run this ATM.